CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.2.3 (2019-05-03)
+
+- Fixed date range picker component to properly represent Angular form control statuses (`dirty`, `pristine`, etc.). [#44](https://github.com/blackbaud/skyux-datetime/pull/44)
+- Fixed date range picker component to properly use default calculators when `calculatorIds` is set to falsy value. [#43](https://github.com/blackbaud/skyux-datetime/pull/43)
+- Fixed date picker calendar component to default to Sunday as starting day. [#41](https://github.com/blackbaud/skyux-datetime/pull/41)
```

Comment:
Sorry, when I commented about date picker in that other thread, I was referring to this line. 

Switch to datepicker calendar component

Reply:
Done!